### ISSUE TITLE
Metadata information such as TTL and maxIdle time are not replicated via WAN replication

### ISSUE THREAD

**Issue by zdehz** [hazelcast/hazelcast#21340]
**Describe the bug**
Metadata information such as **TTL** and **maxIdle** time are not replicated via WAN replication

**Expected behavior**
As a user when I set TTL or maxIdle time of an entry in a WAN replicated map I expect to see those values in a target cluster.

**To Reproduce**

ClusterA config
```
<map name="metadataTestMap">
    <max-idle-seconds>30</max-idle-seconds>
    <time-to-live-seconds>60</time-to-live-seconds>
    <wan-replication-ref name="my-wan-replication" />
</map
```
ClusterB config
```
<map name="metadataTestMap">
</map
```
Below code runs on ClusterA
```
        metadataTestMap.put(1, 1);
        metadataTestMap.put(2, 2);
        metadataTestMap.put(3, 3, 120, SECONDS, 90, SECONDS);
        metadataTestMap.put(4, 4, 150, SECONDS, 150, SECONDS);
```
after 30 seconds the result from both maps;
```
ClusterA [4, 3]
ClusterB [2, 1, 4, 3]
```
after 150 seconds the result from both maps;
```
ClusterA []
ClusterB [2, 1, 4, 3]

**Additional context**
JDK 11, Hazelcast Platform 5.1

Test config for Cluster A;
[hazelcast.xml.txt](https://github.com/hazelcast/hazelcast/files/8595352/hazelcast.xml.txt)

**Comment by sertugkaya** [comment]
this is a customer affecting critical issue which customer was promised a fix by 5.3 release.

**Comment by sertugkaya** [comment]
per @ahmetmircik this should be fixed in 5.2: https://github.com/hazelcast/hazelcast/pull/22294
